src/library/Uno.Themes.WinUI.Markup/Theme.LightWeight/Themes.LightWeightStyles.HyperlinkButton.cs

```diff
@@ -0,0 +1,100 @@
+
+						[ResourceKeyDefinition(typeof(Brush), "HyperlinkButtonBackgroundPointerOver")]
+						public static ResourceValue<Brush> PointerOver => new("HyperlinkButtonBackgroundPointerOver", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "HyperlinkButtonBackgroundPressed")]
+						public static ResourceValue<Brush> Pressed => new("HyperlinkButtonBackgroundPressed", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "HyperlinkButtonBackgroundDisabled")]
+						public static ResourceValue<Brush> Disabled => new("HyperlinkButtonBackgroundDisabled", true);
+					}
+
+					public static class BorderBrush
+					{
+						[ResourceKeyDefinition(typeof(Brush), "HyperlinkButtonBorderBrush")]
+						public static ResourceValue<Brush> Default => new("HyperlinkButtonBorderBrush", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "HyperlinkButtonBorderBrushPointerOver")]
+						public static ResourceValue<Brush> PointerOver => new("HyperlinkButtonBorderBrushPointerOver", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "HyperlinkButtonBorderBrushPressed")]
+						public static ResourceValue<Brush> Pressed => new("HyperlinkButtonBorderBrushPressed", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "HyperlinkButtonBorderBrushDisabled")]
+						public static ResourceValue<Brush> Disabled => new("HyperlinkButtonBorderBrushDisabled", true);
+					}
+
+					public static class Opacity
+					{
+						[ResourceKeyDefinition(typeof(Brush), "HyperlinkButtonBackgroundOpacity")]
+						public static ResourceValue<Brush> Default => new("HyperlinkButtonBackgroundOpacity", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "HyperlinkButtonBackgroundOpacityPointerOver")]
+						public static ResourceValue<Brush> PointerOver => new("HyperlinkButtonBackgroundOpacityPointerOver", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "HyperlinkButtonBackgroundOpacityPressed")]
+						public static ResourceValue<Brush> Pressed => new("HyperlinkButtonBackgroundOpacityPressed", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "HyperlinkButtonBackgroundOpacityDisabled")]
+						public static ResourceValue<Brush> Disabled => new("HyperlinkButtonBackgroundOpacityDisabled", true);
+					}
+
+					public static class Typo
+					{
+						[ResourceKeyDefinition(typeof(FontFamily), "HyperlinkButtonFontFamily")]
+						public static ResourceValue<FontFamily> FontFamily => new("HyperlinkButtonFontFamily", true);
+
+						[ResourceKeyDefinition(typeof(double), "HyperlinkButtonFontSize")]
+						public static ResourceValue<double> FontSize => new("HyperlinkButtonFontSize", true);
+					}
+				}
+
+				public static class Secondary
+				{
+					public static class Foreground
+					{
+						[ResourceKeyDefinition(typeof(Brush), "SecondaryHyperlinkButtonForeground")]
+						public static ResourceValue<Brush> Default => new("SecondaryHyperlinkButtonForeground", true);
+					}
+				}
```

Comment:
@agneszitte @Xiaoy312 not sure about this part, is there a better way (or correct way) for this secondary brush? there's only one: https://github.com/unoplatform/Uno.Themes/blob/66d109c155259d176108a81e9ff1653daeb672a6/src/library/Uno.Material/Styles/Controls/v2/HyperlinkButton.xaml#L35

Reply:
the 2ndary style is not really implemented fully
just nuke that resource, and inline value on the setter

Reply:
I am not following @Xiaoy312, secondary style is the same as the default main style with only a different foreground at the moment in XAML (cc @kazo0)
https://github.com/unoplatform/Uno.Themes/blob/e0e260d6415f95966b95789be4ca79c7e72fb3f7/src/library/Uno.Material/Styles/Controls/v2/HyperlinkButton.xaml#L162

Reply:
If we decide to remove the Secondary style it should be done in a separated PR first for the XAML then C# Markup should match

Reply:
im not saying to nuke the 2ndary style, just its resources and inline the values

Reply:
lets fast-track this #1092

Reply:
fixed